MULTISITE.md

```diff
@@ -0,0 +1,150 @@
+# 🌐 Configuring Multisite with Skela
+
+> **NOTE:** Currently, Skela only supports Multisite with **subdirectories**. If you must use subdomains, Skela is the wrong tool for you.
```

Comment:
@mgburns Nicolette and I tried setting up subdomains with multisite, but couldn't figure out how to get it to work. I'm not sure if something like `site1.test.ups.dock` would work with ups dock 🤔

Reply:
I was able to get subdomain installs working with the following `VIRTUAL_HOST` value in my `.env` file:

```
VIRTUAL_HOST=YOURSITE.ups.dock,*.YOURSITE.ups.dock
```

Though it did expose a bug with the Ups Dock Dashboard:
![image](https://user-images.githubusercontent.com/1508604/74070744-e0fdaa80-49cf-11ea-8fdf-04122b0a17f0.png)

Issue logged here:
https://github.com/Upstatement/ups-dock/issues/18